### PR TITLE
[MPMD-311] - Improve excludeFileFromFailure docs

### PR DESCRIPTION
**Pull request by TobiTenno**
Improve the documentation around `exclude-cpd.properties` file formatting

https://issues.apache.org/jira/browse/MPMD-311

### CODE REVIEW

File: src/site/apt/examples/violation-exclusions.apt.vm [hunk 1]
```diff
@@ -63,9 +63,15 @@ Violation Exclusions
 +-----+
 
   For cpd check, you can exclude classes to not verify. The properties file must have the following format:
+  
+  The differences here are that:
```
Comment (adangel) [inline src/site/apt/examples/violation-exclusions.apt.vm:67]: I'd rephrase this a bit:

```
For cpd check, you can exclude classes to not verify. Classes can be specified in two ways:

  [[1]] Comma-separated: org.apache.maven.ClassA and org.apache.maven.ClassB contain duplicated code across classes.
  [[2]] Single class per line: org.apache.maven.ClassC and org.apache.maven.ClassD do not share duplicate code, but have duplicated code internally.

The properties file for cpd must have the following format:
```

**Review by adangel — APPROVED** [review]

**Comment by adangel** [comment]
Thanks! This will be part in the next version

File: src/site/apt/examples/violation-exclusions.apt.vm (hunk 1 above)
Comment (TobiTenno) [reply src/site/apt/examples/violation-exclusions.apt.vm:67]: Looks awesome, thank you very much!

**Comment by jira-importer** [comment]
Resolve #282

**Comment by jira-importer** [comment]
Resolve #282
